### PR TITLE
Adding a nitrogen surface reaction library

### PR DESCRIPTION
**Pull request by mazeau**
These came from Micro-kinetic modeling of NH3 decomposition on Ni and
its application to solid oxide fuel cells
From Deutschmann et al
doi: 10.1016/j.ces.2011.07.007

This the same as #388, but changed the branch name so the rmg-tests will pass

### CODE REVIEW

File: input/kinetics/libraries/Surface/Nitrogen/dictionary.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+5 H  u0 p0 c0 {1,S}
+
+NO_X
+1 O u0 p2 c0 {2,D}
+2 N u0 p1 c0 {1,D} {3,S}
+3 X u0 p0 c0 {2,S}
+
+O_X
+1 O u0 p2 c0 {2,D}
+2 X u0 p0 c0 {1,D}
```
Comment (nateharms) [inline input/kinetics/libraries/Surface/Nitrogen/dictionary.txt:32]: Should there be a new line at the end of a dictionary? Wondering on the basis of formatting.

File: input/kinetics/libraries/Surface/Nitrogen/reactions.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# encoding: utf-8
+
+name = "Nitrogen"
+shortDesc = u""
+longDesc = u"""
+Nitrogen surface reactions that are too specific to make families for.
+"""
+
+entry(
+    index = 2,
```
Comment (nateharms) [inline input/kinetics/libraries/Surface/Nitrogen/reactions.py:11]: I'm kind of confused by the indexing for the reactions here. @mazeau could you explain this a little?

**Review by nateharms — CHANGES_REQUESTED** [review]
You're basically adding a few reactions for adsorption of nitrogen species to surfaces. I have a few comments but overall, looks good

File: input/kinetics/libraries/Surface/Nitrogen/reactions.py (hunk 2 above)
Comment (mazeau) [reply input/kinetics/libraries/Surface/Nitrogen/reactions.py:11]: index relates to the reaction number in the paper

File: input/kinetics/libraries/Surface/Nitrogen/dictionary.txt (hunk 1 above)
Comment (mazeau) [reply input/kinetics/libraries/Surface/Nitrogen/dictionary.txt:32]: I can add one

**Review by nateharms — APPROVED** [review]
Looks good to me. You addressed my one comment.

**Pull request merged**